### DISCUSSION TITLE
[example/design-system]: default github action is not work error Couldn't find any versions for "eslint-config-neo" that matches "workspace:*"

### DISCUSSION THREAD

**Discussion by vaynevayne** [vercel/turborepo#5032] [Help] [answered]
https://github.com/vaynevayne/neo/actions/runs/5021152885/jobs/9003257973

**Comment by JasperAlexander** [comment]
Your link isn't working.

**Comment by anthonyshew** [comment, marked as answer]
Going to close for staleness here. The link doesn't go anywhere and we don't have a reproduction. Feel free to reopen if you've got one!
